test/integration/run_int_test.go

```diff
@@ -203,13 +208,63 @@ func (suite *RunIntegrationTestSuite) TestRun_DeprecatedLackingLanguage() {
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
 
-	suite.createProjectFile(ts)
+	suite.createProjectFile(ts, 3)
 
 	cp := ts.Spawn("run", "helloWorld")
 	cp.Expect("DEPRECATION", 5*time.Second)
 	cp.Expect("Hello", 5*time.Second)
 }
 
+func (suite *RunIntegrationTestSuite) TestRun_MultipleLanguagesPython2() {
+	if runtime.GOOS == "windows" && e2e.RunningOnCI() {
+		suite.T().Skip("Windows CI does not support ctrl-c events")
+	}
+
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+	suite.createProjectFile(ts, 2)
+
+	ts.LoginAsPersistentUser()
+	defer ts.LogoutUser()
+
+	cp := ts.SpawnWithOpts(
+		e2e.WithArgs("activate"),
+		e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
+	)
+	cp.Expect("Activating state: ActiveState-CLI/Python2")
+	cp.WaitForInput(120 * time.Second)
+
+	cp.SendLine(fmt.Sprintf("%s run helloWorldMultiple", cp.Executable()))
+	cp.Expect("2")
+	cp.SendLine("exit")
+	cp.ExpectExitCode(0)
+}
+
+func (suite *RunIntegrationTestSuite) TestRun_MultipleLanguagesPython3() {
+	if runtime.GOOS == "windows" && e2e.RunningOnCI() {
+		suite.T().Skip("Windows CI does not support ctrl-c events")
+	}
+
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+	suite.createProjectFile(ts, 3)
+
+	ts.LoginAsPersistentUser()
+	defer ts.LogoutUser()
+
+	cp := ts.SpawnWithOpts(
+		e2e.WithArgs("activate"),
+		e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
+	)
+	cp.Expect("Activating state: ActiveState-CLI/Python3")
+	cp.WaitForInput(120 * time.Second)
+
+	cp.SendLine(fmt.Sprintf("%s run helloWorldMultiple", cp.Executable()))
+	cp.Expect("3")
+	cp.SendLine("exit")
+	cp.ExpectExitCode(0)
+}
```

Comment:
I feel like we can cover this with our existing activation tests, no need to create new ones just to make this one single assertion.

ie. do this assertion in one of the tests we already have for running activations.